client/Components/CurrencyChart.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+  HorizontalGridLines,
+  VerticalGridLines,
+  XAxis,
+  XYPlot,
+  YAxis, LineMarkSeries
+} from 'react-vis';
+
+class CurrencyChart extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+    }
+  }
+
+  componentDidMount() {
+    setInterval(() => {
+      fetch('https://min-api.cryptocompare.com/data/pricemulti?fsyms=BTC&tsyms=USD&api_key=c3928dbf8fd1c2f62b730f46a5fff3d1ba38535ecd0609cb68c95c31d92edd25').then(res => res.json())
```

Comment:
Is this API key sensitive at all?
As a general rule, api keys shouldn't be in our version control, probably.

Reply:
whoops - will fix